haiku-games/hexvexed/hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION=""HexVexed is a maddening addictive and easy to play puzzle game:
+Arrange hexagonal tiles into a grid so that the numbers and colours match
+where they touch. This game is a version DarkWyrm's BeVexed, but instead
+of squares, the tiles a hexagonal shaped. It's 1.5x the fun!
```

Comment:
Can you please change DESCRIPTION to:
```
DESCRIPTION=""HexVexed is a maddening addictive and easy to play puzzle game: \
Arrange hexagonal tiles into a grid so that the numbers and colours match \
where they touch. This game is a version DarkWyrm's BeVexed, but instead \
of squares, the tiles a hexagonal shaped. It's 1.5x the fun!"
```
(Yep, I know, the backslashes were missing in the last suggestion...)
The reason why we prefer to have backslashes is to have a nice dynamic layout, instead of random line feeds in HaikuDepot.